### PR TITLE
Change the python project manager

### LINKED ISSUE #43: Moving from poetry to hatch

### PR DESCRIPTION
**Pull request by larnaa**
Close #43

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,8 +16,11 @@ jobs:
...
-      - name: Install Poetry
-        uses: snok/install-poetry@v1
+      - name: Ensure latest pip
+        run: python -m pip install --upgrade pip
 
-      - name: Makefile
-        run: make
\ No newline at end of file
+      - name: Install Hatch
+        run: pip install hatch
```
Comment (droserasprout) [inline .github/workflows/ci.yml:23]: Maybe use pipx instead? It's also preinstalled in GH runner images.

File: pyproject.toml [hunk 2]
```diff
@@ -1,70 +1,102 @@
-[tool.poetry]
-name = "cluecoins"
-version = "0.0.0"
-description = "A CLI tool to manage the database of Bluecoins, an awesome budget planner for Android"
-license = "MIT"
-authors = [
-    "Lev Gorodetskiy <github@droserasprout.space>"
```
Comment (droserasprout) [inline pyproject.toml:7]: 😭

File: README.md [hunk 3]
```diff
@@ -10,22 +10,18 @@
...
 
-To manage development environment you need GNU Make and [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer) installed.  
-Run `poetry build` to make a Python wheel package. Run `make` to run an entire CI pipeline: install dependencies, run linters and tests.
-
-## Usage
-
-Run `cluecoins` command and follow the instructions.
-
-## Console interface
+To use the Cluecoins you need [Hatch](https://hatch.pypa.io/1.5/install/) installed.
```
Comment (droserasprout) [inline README.md:19]: Not exactly. `pip install -e .` will also work.

File: README.md [hunk 4]
```diff
@@ -10,22 +10,18 @@
 
 A tool to manage the database of [Bluecoins](https://www.bluecoinsapp.com/), a finance manager for Android.
 
+Cluecoins can be used in two ways: by CLI or TUI.
```
Comment (droserasprout) [inline README.md:13]: Please mention root usage in tui mode lol

Comment (droserasprout) [reply README.md:13]: And backup paths (!)

**Review by droserasprout — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (larnaa) [reply pyproject.toml:7]: Ahh, I got you back in the author's block

**Pull request merged**